### PR TITLE
Add  code of conduct page.

### PR DESCRIPTION
**Pull request by Carreau**
Unlike other pages is does not have an `.html` url ending, you "just"
visit /conduct/ url to see the page.

The content should of course be filled, and a link added in the header

### CODE REVIEW

**Comment by Carreau** [comment]
CC @willingc, @ruv7 reminded me that this was not done, so I did that quickly. Can you have a look ? Thanks !

File: conduct/index.md [hunk 1]
```diff
@@ -0,0 +1,154 @@
...
+---
+
+
+Jupyter Code of Conduct can be found on the [Jupyter/Governance GitHub
+Repository](https://github.com/jupyter/governance). For convenience it has been
+replicated here. 
+
+In case of difference,  the version on the [Jupyter/Governance GitHub
+Repository](https://github.com/jupyter/governance) is the canonical one, and
+the one present here should be updated accordigly.
```
Comment (willingc) [inline conduct/index.md]: accordingly

Comment (Carreau) [inline conduct/index.md]: fixed.

**Comment by Carreau** [comment]
mail to `conduct@jupyter` are FWD to the project gmail address and whoever checks it. I can FWD to more people if requested, or maybe "delegate" access to the jupyter gmail account.

Fernando sent a test mail on Dec 23rd, so I assume he monitors as well.

**Comment by willingc** [comment]
Thanks @Carreau for getting this done. Merging.

**Pull request merged**